services/checks/checks.go

```diff
@@ -451,8 +452,36 @@ func (s *Service) minPMMAgentVersion(t check.Type) *version.Parsed {
 	}
 }
 
-// executeChecks runs all available checks for all reachable services.
-func (s *Service) executeChecks(ctx context.Context) error {
+func (s *Service) filterChecks(checks []check.Check, disable, enable []string) []check.Check {
```

Comment:
a comment explaining this would be nice since we will have 2 filterChecks functions after PMM-7336 is merged
something like 
``` 
// filterChecks  filters the list of checks received from the UI to be executed from the checks disabled in settings
```

Reply:
Well, there is only one function now, but it's not so easy to understand.